site/content/contribute/more-info/server/rest-api.md

```diff
@@ -85,8 +95,11 @@ The most important part of this process is to make sure the new endpoint works c
     - Requests with insufficient permissions return a **403 Forbidden** status code (for endpoints requiring permission).
     - Requests to non-existent resources or URLs return a **404 Not Found** status code.
 
-Returning the correct error code might require investigation in the {{< newtabref href="https://github.com/mattermost/mattermost/tree/master/server/channels/app" title="app" >}} or {{< newtabref href="https://github.com/mattermost/mattermost/tree/master/server/channels/store" title="store" >}} packages to find the source of errors. Status codes on errors should be set at the creation of the error.
+Returning the correct error code might require investigation in the [app](https://github.com/mattermost/mattermost/tree/master/server/channels/app) or [store](https://github.com/mattermost/mattermost/tree/master/server/channels/store) to find the source of errors. Status codes on errors should be set at the creation of the error. 
```

Comment:
Did you mean "incorrect error code"?

Reply:
I admit I didn't parse what the original text meant in great detail, but looking at it closely now, it seems like it does mean returning the "correct" error code, as the context sugests one of 403, 404, etc. Thoughts?

Reply:
Okay, I interpreted it incorrectly as I thought that like: "if the API error code is not correct, it requires investigation", but I think it's more about how to find out accurate error codes for the API.